src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -18,8 +18,9 @@
 import json
 import satellite_images_nso._manipulation.nso_manipulator as nso_manipulator
 import satellite_images_nso._nso_data_extraction.nso_api as nso_api
+from shapely.ops import unary_union
 
-
+# TODO: Make a decision about the logging.
```

Comment:
I would just choose one....having double logging lines is extremely cumbersome and pointless...